evm/src/common.rs

```diff
@@ -6,9 +6,9 @@ use std::u64;
 #[inline]
 pub fn to_word_size(size: u64) -> u64 {
     if size > u64::MAX - 31 {
-        return u64::MAX >> 5 + 1;
+        return u64::MAX >> (5 + 1);
```

Comment:
Is it right?

Reply:
No. It's a bug written by me. Use `(u64::MAX >> 5) + 1` instead plz.